CHANGELOG.md

```diff
@@ -2,8 +2,17 @@
 
 ## [Unreleased][]
 
-[Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.26.1...HEAD
+[Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.26.2...HEAD
 
+## [0.26.1][] - 2021-12-13
+
+[0.26.2]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.26.1...0.26.2
+
```

Comment:
We prefer that PRs do not set the versions. Simply remove these lines and it'll be fine :)